src/js/controller/controller.js

```diff
@@ -255,6 +255,8 @@ define([
                 if (_model.get('state') === states.ERROR) {
                     _model.set('state', states.IDLE);
                 }
+                _model.set('preInstreamState', 'instream-idle');
```

Comment:
Where can we put it in instream-adapter so that all changes to 'preInstreamState' are done in insteam?